cumulus/parachains/runtimes/starters/shell/src/lib.rs

```diff
@@ -82,8 +82,8 @@ impl_opaque_keys! {
 /// This runtime version.
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("shell"),
```

Comment:
`create_runtime_str` macro should stay and annotated as deprecated.

Reply:
I was thinking about that and to also make `RuntimeString` a type alias to the `Cow`, but wasn't sure if it was worth it. Seemed like a trivial change to upgrade, also required re-export of the hidden `Cow` from the crate, which I also didn't like at all,  and if `RuntimeString` is removed then it is still a breaking change one way or the other.

If still desirable, should I also keep re-exports in `sp-version`?

Reply:
@bkchr I'd like to get clarity here so I can update PR with corresponding changes